CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### New Features and Enhancements
 
+The default value of `CMAKE_CUDA_ARCHITECTURES` is no longer set to `70` and is
+now determined automatically by CMake. Users are encouraged to override the
+default value as it varies across compilers and compiler versions.
```

Comment:
Are you saying that the now-automatically-determined value of `CMAKE_CUDA_ARCHITECTURES` *should* be changed by users, or that the previous default value needed to be changed by users?  I recommend that this sentence be revised slightly to clarify.

Reply:
Both, the old default was only correct for Volta GPUs and users would need to change it when targeting other GPUs. The automatically determined value might be correct but it is recommended that users specify the value.